kubeflow/hack/create_context.sh

```diff
@@ -6,24 +6,17 @@
 # PROJECT=<PROJECT> REGION=<region> NAME=<NAME>
 #
 # TODO(jlewi): Support zonal clusters as well
-set -x 
+set -x
 
-# Default namespace to kubeflow
-NAMESPACE=${NAMESPACE:-kubeflow}
-echo Checking if context ${NAME} exists 
-
-kubectl config use-context ${NAME}
-
-RESULT=$?
-
-if [ ${RESULT} -eq 0 ]; then
-echo kubeconfig context ${NAME} already exists
-exit 0
-fi
+# Delete kubeconfig context if exist already
+kubectl config delete-context ${NAME}
```

Comment:
Recommend opt out one command explicitly sort of like
```
kubectl config delete-context ${NAME} || echo "OK - context ${NAME} does not exist"
```
It's also important to let people know if there is an error message there, it's expected. This helps when people debug other problems

Reply:
Updated error message based on the exact output
```zsh
error: cannot delete context ${NAME}, not in path/to/config
```